src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -214,5 +214,18 @@ private async Task<bool> IsWorkerReadyForRequest()
                 return false;
             }
         }
+
+        public async Task PingAsync()
+        {
+            string requestUri = new UriBuilder(WorkerConstants.HttpScheme, WorkerConstants.HostName, _httpWorkerOptions.Port).ToString();
+            try
+            {
+                await _httpClient.GetAsync(requestUri);
+            }
+            catch (Exception ex)
+            {
+                _logger.LogDebug($"Pinging url {requestUri} resulted in exception", ex);
```

Comment:
It is worth logging response code if request succeeds. This will help is determine with customHandler do return 200 or not on the root.